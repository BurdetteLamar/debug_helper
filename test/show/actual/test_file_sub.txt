---
DebugHelperTest::FileSub (message='test_file_sub'):
  DebugHelperTest::FileSub.executable?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  DebugHelperTest::FileSub.exist?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): true
  DebugHelperTest::FileSub.ftype("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): file
  DebugHelperTest::FileSub.pipe?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  DebugHelperTest::FileSub.readable?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): true
  DebugHelperTest::FileSub.setgid?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  DebugHelperTest::FileSub.setuid?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  DebugHelperTest::FileSub.socket?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  DebugHelperTest::FileSub.symlink?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  DebugHelperTest::FileSub.writable?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): true
