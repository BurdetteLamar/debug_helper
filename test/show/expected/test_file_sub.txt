---
DebugHelperTest::FileSub (message='test_file_sub'):
  executable?: false
  exist?: true
  ftype: file
  pipe?: false
  readable?: true
  setgid?: false
  setuid?: false
  socket?: false
  symlink?: false
  writable?: true
